products/ubuntu2004/profiles/stig.profile

```diff
@@ -349,7 +349,7 @@ selections:
     - auditd_audispd_configure_remote_server
 
     # UBTU-20-010217 The Ubuntu operating system must immediately notify the SA and ISSO (at a minimum) when allocated audit record storage volume reaches 75% of the repository maximum audit record storage capacity.
-    - var_auditd_space_left=250MB
+    - var_auditd_space_left=25%
     - var_auditd_space_left_action=email
     - auditd_data_retention_space_left_action
     - auditd_data_retention_space_left
```

Comment:
```suggestion
    - var_auditd_space_left_percentage=25
    - var_auditd_space_left_action=email
    - auditd_data_retention_space_left_action
    - auditd_data_retention_space_left_percentage
```

I believe these are the changes you are looking for

Reply:
oh, that's great, I completely missed this new rule, thanks!
it is rebased now